app/uk/gov/hmrc/nationalimportdutyadjustmentcentrefrontend/navigation/Navigator.scala

```diff
@@ -16,72 +16,25 @@
-    P(CustomsDutyRepaymentPage, makeclaim.routes.DutyRepaymentController.onPageLoadCustomsDuty, hasDutyType(Customs)),
-    P(ImportVatRepaymentPage, makeclaim.routes.DutyRepaymentController.onPageLoadImportVat, hasDutyType(Vat)),
-    P(OtherDutyRepaymentPage, makeclaim.routes.DutyRepaymentController.onPageLoadOtherDuty, hasDutyType(Other)),
-    P(ClaimReasonPage, makeclaim.routes.ClaimReasonController.onPageLoad, always),
-    P(UploadPage, makeclaim.routes.UploadFormController.onPageLoad, hasNoUploads),
-    P(UploadSummaryPage, makeclaim.routes.UploadFormSummaryController.onPageLoad, hasUploads),
-    P(ContactDetailsPage, makeclaim.routes.ContactDetailsController.onPageLoad, always),
-    P(AddressPage, makeclaim.routes.AddressController.onPageLoad, always),
-    P(ImporterHasEoriNumberPage, makeclaim.routes.ImporterHasEoriController.onPageLoad, isRepresentative),
-    P(ImporterEoriNumberPage, makeclaim.routes.ImporterEoriNumberController.onPageLoad, enterImporterEori),
-    P(ImporterContactDetailsPage, makeclaim.routes.ImporterDetailsController.onPageLoad, isRepresentative),
-    P(RepayToPage, makeclaim.routes.RepayToController.onPageLoad, isRepresentative),
-    P(BankDetailsPage, makeclaim.routes.BankDetailsController.onPageLoad, always),
-    P(CheckYourAnswersPage, makeclaim.routes.CheckYourAnswersController.onPageLoad, always),
-    P(ConfirmationPage, makeclaim.routes.ConfirmationController.onPageLoad, always)
-  )
+  private lazy val reversePageOrder = pageOrder.reverse
 
-  private val reversePageOrder = pageOrder.reverse
-
-  def nextPage(currentPage: Page, userAnswers: CreateAnswers): Call =
+  def nextPage(currentPage: Page, userAnswers: T): Call =
     viewFor(pageOrder, nextPageFor(pageOrder, currentPage, userAnswers)).getOrElse(pageOrder.head.destination())
 
-  def previousPage(currentPage: Page, userAnswers: CreateAnswers): NavigatorBack =
+  def previousPage(currentPage: Page, userAnswers: T): NavigatorBack =
     NavigatorBack(viewFor(pageOrder, nextPageFor(reversePageOrder, currentPage, userAnswers)))
 
-}
-
-protected trait Conditions {
-  protected val always: CreateAnswers => Boolean = (_: CreateAnswers) => true
-
-  protected val hasDutyType: ReclaimDutyType => CreateAnswers => Boolean = (dutyType: ReclaimDutyType) =>
-    _.reclaimDutyTypes.contains(dutyType)
-
-  protected val hasNoUploads: CreateAnswers => Boolean = _.uploads.isEmpty
-
-  protected val hasUploads: CreateAnswers => Boolean = _.uploads.nonEmpty
-
-  protected val isRepresentative: CreateAnswers => Boolean = _.isRepresentative
-
-  protected val enterImporterEori: CreateAnswers => Boolean = (answers: CreateAnswers) =>
-    isRepresentative(answers) && answers.doesImporterHaveEori
-
-}
-
-protected trait Ordering {
-
-  protected val nextPageFor: (Seq[P], Page, CreateAnswers) => Option[Page] = (pages, currentPage, userAnswers) =>
+  protected val nextPageFor: (Seq[P], Page, T) => Option[Page] = (pages, currentPage, userAnswers) =>
```

Comment:
`nextPageFor` can be private

```suggestion
  private val nextPageFor: (Seq[P], Page, T) => Option[Page] = (pages, currentPage, userAnswers) =>
```